lib/hammer_cli_foreman_puppet/environment_name_mapping.rb

```diff
@@ -0,0 +1,20 @@
+module HammerCLIForemanPuppet
+  module EnvironmentNameMapping
```

Comment:
Loading of the plugin fails with 
[ERROR 2021-06-23T09:16:21 Modules] <NameError> uninitialized constant HammerCLIForemanPuppet::PuppetEnvironment::EnvironmentNameMapping
	/home/vagrant/hammer-cli-foreman-puppet/lib/hammer_cli_foreman_puppet/environment.rb:3:in <class:PuppetEnvironment>
`